packages/opc-header/dist/index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>opc-header</title>
+    <script src="opc-header.js"></script>
+    <style>
+        body {
+            display: flex;
+            flex-direction: column;
+            justify-content: center;
+            align-items: center;
+            height: 100vh;
+        }
+        .test__styles {
+            font-size: 1rem;
+        }
+    </style>
+</head>
+<body>
+    <opc-header style="--opc-header--width: 1200px;">
+        <div slot="breadcrumb-slot">
```

Comment:
opc-header have breadcrumb slot???? also dont name slots like this

Reply:
can you plz suggest how do i name it.

Reply:
@anjneeksharma Follow the best practices for writing custom elements, and some naming conventions.
- https://developers.google.com/web/fundamentals/web-components/best-practices

Also, use BEM for css classes.
- http://getbem.com/introduction/
- https://css-tricks.com/bem-101/